src/main/java/frc/robot/Constants.java

```diff
@@ -5,132 +5,141 @@
-  public static final double WHEELBASE =
-      1.1333; // the ratio between the width and the length is around 3:4
-  // public static final double TRACKWIDTH = 1;
-  // public static final double WHEELBASE = 1;
-
-  public static final double SPEED_DIVISOR =
-      1; // what the max speed should be divided by, 1 is max power
-  public static final double ARM_SPEED = 0.005; // speed of the arms when adjusting manually
-
-  // CHARGE STATION COMMUNITY DISTANCE:
-  public static final double encoderToBlueChargeDistance = 96.4694981;
-  public static final double encoderToRedChargeDistance = 99;
-  // public static final double encoderToChargeDistance = 45.4694981;
-
-  public static final double encoderCenterCommunity = 100;
-  public static final double encoderToOutsideCommunityDistance = 87.30208217;
-  // 1.832716884 is the number of inches per 1 encoder value
-  // ~80 (plus offset) to the center of the charge station for robot
-  // ~160 is the distance to leave the community plus some extra cushion
+
+  public static final class CanCoderID {
+    public static final int GYRO = 0;
+    public static final int FL = 43;
+    public static final int FR = 44;
+    public static final int BL = 42;
+    public static final int BR = 41;
+    public static final int SMALL_ARM = 22;
+    public static final int BIG_ARM = 21;
+  }
+
+  public static final class AngleID {
+    public static final int FL = 15;
+    public static final int FR = 16;
+    public static final int BL = 11;
+    public static final int BR = 18;
+  }
+
+  public static final class MotorID {
+    public static final int FL = 17;
+    public static final int FR = 12;
+    public static final int BL = 13;
+    public static final int BR = 14;
+    public static final int BIG_ARM = 6;
+    public static final int BIG_ARM_2 = 7;
+    public static final int SMALL_ARM = 31;
+    public static final int Compressor = 1;
+  }
+
+  public static final class BigArmPosition {
+    public static final double HIGH = 0.578857;
+    public static final double MEDIUM = 0.437500;
+    public static final double PICKUP = 0.392578;
+    public static final double DEFAULT = 0.386475;
+    public static final double SLIDER = 0.595092; // Fine tune
+    public static final double PICKUPCONE = 0.47295;
+  }
+
+  public static final class isInverted {
+    // This is for moters
+    public static final boolean FL = false;
+    public static final boolean FR = true;
+    public static final boolean BL = false;
+    public static final boolean BR = true;
+  }
+
+  public static final class Canbus {
+    public static final String CANBUS = "rio";
+  }
+
+  public static final class Canivore {
+    public static final String CANIVORE = "drivetrain";
+  }
+
+  public static final class Sensor {
+    public static final String CANIVORE = "drivetrain";
+    public static final int CUBE_CHANNEL = 2;
+    public static final int CONE_CHANNEL = 3;
+  }
+
+  public static final class Gamepad {
```

Comment:
Ditto of above, go for `Gamepad.Controller.DRIVE` and `Gamepad.Axis.LEFT_X` and `Gamepad.Button.A` using nested classes